api/spec/v1.yaml

```diff
@@ -2100,6 +2104,34 @@ components:
       description: |
         A runtime transaction.
 
+    RuntimeTransactionEncryptionEnvelope:
+      type: object
+      required: [format, public_key, data_nonce, data, result_nonce, result]
```

Comment:
Just thinking out loud: When we're returning a list of txs, we might want to keep the response a little more lightweight and not return anything but the encryption format; the body and result are on the hefty side. Some options:
- Ignore this possibility, keep everything as-is. If we ever need to implement these lighter responses, we'll break backwards compatilibiltiy and mark all fields except `format` as optional; or we'll introduce another field `is_encrypted: bool` parallel to the envelop field, and drop the entire envelope field in list queries.
- Mark only `format` as required, so that if we start dropping the other fields in the future, we're not breaking the API.
- Move `format` one level up, so we'd have `encryption_format` (could be then extended to non-evm txs too) and `evm_encrypted_contents`. In list queries, we could omit `evm_encrypted_contents` in the future if we wanted.

We should realistically probably go for #2? Minimal and quick change. A little dirty because the resulting API is sloppy (no guarantees of when the fields will/won't be present), but still fine IMO.

On the topic of optional fields: line 2067 should now probably be `May be null if the transaction was malformed or encrypted.`

Reply:
:+1: also after what we discussed about Cipher, having `format` a level higher would be potentially confusing if we ever wanted to expand to Cipher, or to other confidential runtimes that encrypt a different subset of the runtime's tx.